cinema/views.py

```diff
@@ -0,0 +1,56 @@
+from django.http import HttpResponse
+from django.shortcuts import get_object_or_404
+from rest_framework import status
+from rest_framework.decorators import api_view
+from rest_framework.response import Response
+
+from cinema.models import Movie
+from cinema.serializers import MovieSerializer
+
+
+@api_view(["GET", "POST"])
+def movie_list(request):
+    if request.method == "GET":
+        movies = Movie.objects.all()
+        serializer = MovieSerializer(movies, many=True)
+        return Response(serializer.data, status=status.HTTP_200_OK)
+
+    serializer = MovieSerializer(data=request.data)
+    if serializer.is_valid(raise_exception=True):
```

Comment:
if statement is redundant
```suggestion
    serializer.is_valid(raise_exception=True):
```

Reply:
Fix in all places